docs/07. Manuales de usuario/7.3. Software Reviewer Guideline - Sprint 3.mdx

```diff
@@ -0,0 +1,350 @@
+# Software Reviewer Guidelines - Sprint 3
+
+# Introduccion
+
+En el documento vigente se definirán las guías necesarias para probar cada una de las funcionalidades que las aplicaciones Web de Cirio y Costal y ACAT proporcionas. La guía estará definida para usuarios candidatos de la aplicación, cuyas actividades difieren en la aplicación de aquellas que realizan los usuarios representantes.
+
+# Información Importante
+
+Existen actualmente dos despliegues de nuestro sistema, uno para ONG que se beneficiará del mismo, debido a las diferencias entre sus necesidades. Se puede acceder a estas a través de los siguientes enlaces:
+- [ACAT](https://frontend-acat-sprint-3.vercel.app/)
+- [CyC](https://frontend-cyc-sprint-3.vercel.app/)
+
+## Navegadores compatibles
+Actualmente solo navegadores basados en la tecnología Chromium tienen soporte. En un futuro se pretende dar soporte para otros navegadores pero actualmente no se puede asegurar el correcto funcionamiento del sistema en navegadores basados en webkit como Safari, o independientes como Firefox.
+
+## Permisos
+En el sistema desarrollado se espera que todos los usuarios sean administradores, por lo que tanto en esta iteración como iteraciones posteriores se espera que cualquier persona con acceso al sistema pueda hacer cualquier operación permitida.
+
+## Información de Contacto
+Si en algún momento durante las pruebas necesita informar de alguna duda o problema, existen las siguientes vías:
+- Correo Electrónico: harmonyltd@outlook.es
+- Formulario: Podrás dejar cualquier tipo de feedback para reportar errores, recomendar cambios o proponer mejoras en el siguiente formulario: [enlace]
```

Comment:
This link is broken. Please fix by inserting this one:

https://docs.google.com/forms/d/e/1FAIpQLSda1exsoZI3jOJIednAizC8H4gx9w8Q2LoRb0lMl1Rdzw3wtA/viewform?usp=sf_link